port/unix/omrsysinfo.c

```diff
@@ -657,22 +657,26 @@ omrsysinfo_processor_has_feature(struct OMRPortLibrary *portLibrary, OMRProcesso
 }
 
 intptr_t
-omrsysinfo_processor_disable_feature(struct OMRPortLibrary *portLibrary, OMRProcessorDesc *desc, uint32_t feature)
+omrsysinfo_processor_set_feature(struct OMRPortLibrary *portLibrary, OMRProcessorDesc *desc, uint32_t feature, BOOLEAN enable)
 {
 	intptr_t rc = -1;
-	Trc_PRT_sysinfo_processor_disable_feature_Entered(desc, feature);
+	Trc_PRT_sysinfo_processor_set_feature_Entered(desc, feature, enable);
 
 	if ((NULL != desc) && (feature < (OMRPORT_SYSINFO_OS_FEATURES_SIZE * 32))) {
 		uint32_t featureIndex = feature / 32;
 		uint32_t featureShift = feature % 32;
 
-		if (OMR_ARE_ALL_BITS_SET(desc->features[featureIndex], 1u << featureShift)) {
+		if ((FALSE == enable) && OMR_ARE_ALL_BITS_SET(desc->features[featureIndex], 1u << featureShift)) {
```

Comment:
I think the way we are clearing bits could be simpler:
```c++
if (enable) {
		desc->features[featureIndex] |= (1u << featureShift);
} else {
		desc->features[featureIndex] &= ~(1u << featureShift);
}
```

Maybe its time to add a macro to `omrcomp.h`
```c++
#define OMR_SET_BITS(value, bits) (value) |= (bits)
#define OMR_CLEAR_BITS(value, bits) (value) &= ~(bits)
```

Reply:
I think i've convinced myself that these macros wouldn't be helpful